index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <header>
+      <a id="logo">Mark.O</a>
+      <nav>
+        <img src="./logos/Union.jpeg" alt="menue bar" id="menue-button">
+      </nav>
+    </header>
+    <section id="intro">
+      <h1>Hi,Im Mark Otuya</h1>
+      <p id="bio">I’m a software developer & Web Designer! I can Build your Dream Web Page on Demand, 
+       Look Below for  some of my work and experience!  
+        Feel free to contact me, Lets work.</p>
+      <h3>Contact Me</h3>
+      
+      <ul id="social">
+        <li><img src="./logos/twitter.png" alt="twitter"></li>
+        <li><img src="./logos/linkedin.png" alt="linkedin"></li>
+        <li><img src="./logos/medium.png" alt="email"></li>
+        <li><img src="./logos/github.png" alt="github"></li>
+        <li><img src="./logos/angelist.png" alt="angel-list"></li>
```

Comment:
- [ ] Please wrap each one of these social media images with a `<a>` tag because it's supposed to be links.
for example:
`<li><a href="your link"><img src="./logos/twitter.png" alt="twitter"></a></li>`